src/Model/Policy.php

```diff
@@ -236,4 +236,23 @@ public function getValuesForFieldInPolicy(string $sec, string $ptype, int $field
 
         return $values;
     }
+
+    /**
+     * gets all values for a field for all rules in a policy of all ptypes, duplicated values are removed.
+     *
+     * @param string $sec
+     * @param int    $fieldIndex
+     *
+     * @return array
+     */
+    public function getValuesForFieldInPolicyAllType(string $sec, int $fieldIndex): array
+    {
+        foreach ($this->items[$sec] as $ptype) {
+            $values = $this->getValuesForFieldInPolicy($sec, $ptype->key, $fieldIndex);
```

Comment:
Reimplement it, refer to: https://github.com/casbin/casbin/blob/master/model/policy.go#L222